CMakeLists.txt

```diff
@@ -320,3 +320,13 @@ endif()
 # ---[ ONNX Interface for Framework Integratin (ONNXIFI)
 add_library(onnxifi INTERFACE)
 target_include_directories(onnxifi INTERFACE onnx)
+
+
+install(TARGETS onnx onnx_proto
+    ARCHIVE DESTINATION lib)
+install(DIRECTORY onnx DESTINATION include
```

Comment:
`onnx` -> `${ONNX_ROOT}/onnx`

Reply:
Should I squash to one commit or commit a new one?